devlab/provision/generate_config.sh

```diff
@@ -10,13 +10,14 @@ error_exit() {
         echo &>2
     fi
 
-    echo "Usage: ${SCRIPT} --cloudferry-path <path>"
+    echo "Usage: ${SCRIPT} --cloudferry-path <path> [<new_file_path>]"
```

Comment:
Please use the same pattern for arguments, even optional. Should be either `${SCRIPT} --cloudferry-path <path> [--destination <path>]` or `${SCRIPT} <cf_path> [<destination_config_path>]`